### PR TITLE
Fixed fontsize parameter in set_xlabel and set_ylabel

### PR DESCRIPTION
**Pull request by Fedorsturov**
EDIT: #5686

### CODE REVIEW

**Comment by taldcroft** [comment]
@Fedorsturov - just a comment on process, please provide a more descriptive PR title that lets people more easily determine if this is relevant to them.

File: CHANGES.rst [hunk 1]
```diff
@@ -90,6 +90,7 @@ API Changes
 - ``astropy.utils``
 
 - ``astropy.visualization``
+  - Created param labelpad in set_xlabel and set_ylabel functions. It is directly passed to set_axislabel as minpad.
```
Comment (astrofrog) [inline CHANGES.rst]: Can you also mention that additional keyword arguments are now also recognized, such as font settings?

Comment (pllim) [inline CHANGES.rst]: Also add PR number in the entry. See other entries for example formatting.

**Comment by astrofrog** [comment]
Continued and closed in #6060

**Pull request closed without merging**